app/javascript/components/analytics/public_health/charts/MonitoreeFlow.js

```diff
@@ -118,17 +157,135 @@ class MonitoreeFlow extends React.Component {
+                      <span className="percentage"></span>
+                    </div>
+                  </td>
+                ))}
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </Col>
+    );
+  }
+
+  renderIsolationToExposureTable(data) {
+    return (
+      <Col xl="12">
+        <div className="analytics-table-header">Moved from Isolation to Exposure</div>
+        <div className="table-responsive">
+          <table className="analytics-table">
+            <thead>
+              <tr className="g-border-bottom text-center header">
+                <th></th>
+                <th>Yesterday</th>
+                <th>Last 7d</th>
+                <th>Last 14d</th>
+                <th>Cumulative</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr className="g-border-bottom">
+                <td className="header">Moved from Isolation to Exposure Workflow</td>
+                {data.map((x, i) => (
+                  <td key={i}>
+                    <div className="count-percent-container">
+                      <span className="number">{x.isolation_to_exposure_total.value}</span>
+                      <span className="percentage align-bottom"></span>
+                    </div>
+                  </td>
+                ))}
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </Col>
+    );
+  }
+
   render() {
     return (
       <Card>
         <Card.Header as="h4" className="text-center">
           Monitoree Flow Over Time (All Records)
         </Card.Header>
         <Card.Body>
-          <Row>{this.tableData.map((data, index) => this.renderWorkflowTable(data, index))}</Row>
-          <div className="text-center text-secondary info-text my-1">
+          <Row>{this.workflowTableData.map((data, index) => this.renderWorkflowTable(data, index))}</Row>
+          <div className="text-center text-secondary info-text mb-4">
+            <i className="fas fa-info-circle mr-1"></i>
+            Cumulative includes incoming and outgoing counts recorded for this jurisdiction (excluding today’s counts). All counts calculated based on UTC time
+            zone.
+          </div>
+          <Row>
+            {this.renderExposureToCaseTable(this.exposureToCaseTableData)}
+            {this.renderIsolationToExposureTable(this.isolationToExposureTableData)}
+          </Row>
+          <div className="text-center text-secondary info-text mb-1">
             <i className="fas fa-info-circle mr-1"></i>
-            Cumulative includes all incoming and outgoing counts ever recorded for this jurisdiction
+            Cumulative includes counts of the monitorees that met the criteria listed in the tables after {
+              this.props.case_development_analytics_start_date
+            }{' '}
+            (excluding today’s counts). All counts calculated based on UTC time zone.
```

Comment:
Can you add a condition on this div to only render if `props.case_development_analytics_start_date` isn't blank?

Taking into account that an instance of the app could be initially deployed at or after version 1.38.

Reply:
****

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1085/commits/3a713d3d7ba105e098cbf811b449fb506dbd5f49